requirements/philu/testing.txt

```diff
@@ -0,0 +1,17 @@
+# TODO: Find replacement of add_or_replace_params method with urllib or some some django builtin package and remove this dependency
+w3lib==1.21.0
+-e git+https://github.com/philanthropy-u/pynodebb.git#egg=pynodebb
+SurveyGizmo==1.2.3
+mandrill==1.0.57
+pyminizip==0.2.1
+channels==1.1.8
+asgi-redis==1.4.3
+imgkit==1.0.1
+pdfkit==0.6.1
```

Comment:
Ideally, we should mock third-party apps like imgkit, pdfkit, and mandrill, etc. We can remove it in the future, once all test are mocked properly